flutter/lib/models/model.dart

```diff
@@ -1810,6 +1906,15 @@ class CanvasModel with ChangeNotifier {
   // scroll offset y percent
   double _scrollY = 0.0;
   ScrollStyle _scrollStyle = ScrollStyle.scrollauto;
+  // tracks whether edge scroll should be active, prevents spurious
+  // scrolling when the cursor enters the view from outside
+  EdgeScrollState _edgeScrollState = EdgeScrollState.inactive;
+  // fallback strategy for when Bump Mouse isn't available
+  late EdgeScrollFallbackState _edgeScrollFallbackState;
+  // to avoid hammering a non-functional Bump Mouse
+  bool _bumpMouseIsWorking = true;
+  // briefly set to true when the canvas detects a resize
+  bool _suppressEdgeScroll = false;
```

Comment:
`_suppressEdgeScroll` is never set, is the comment above wrong?

<img width="1212" height="1074" alt="Image" src="https://github.com/user-attachments/assets/5457ce77-2106-484d-9a9c-06c779361079" />


The `Flutter` extension in VS Code can help the Dart diagnostics. https://dart.dev/tools/diagnostics

Reply:
It's not fundamentally wrong but it's out-of-date, so it's wrong _now_. 🙂